services/cdnjs/cdnjs.service.js

```diff
@@ -6,24 +6,27 @@ const { NotFound } = require('../errors')
 const { addv: versionText } = require('../../lib/text-formatters')
 const { version: versionColor } = require('../../lib/color-formatters')
 
+const cdnjsSchema = Joi.object({
+  version: Joi.string(),
```

Comment:
I think you want `version: Joi.string().required()`.

Reply:
On both CDNJS and Clojars the version key is intentionally optional because `{}` should also pass validation.

Reply:
Could you add a comment in the schema explaining why?